cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -57,7 +57,20 @@ AUTH_TOKEN="{{ '${' }}{{ env_vars.github_token_env_var }}:-}"
 read -r RECEIPT <<EORECEIPT
 {{ receipt | tojson }}
 EORECEIPT
-RECEIPT_HOME="${XDG_CONFIG_HOME:-$HOME/.config}/{{ app_name }}"
+
+# Some Linux distributions don't set HOME
+# https://github.com/astral-sh/uv/issues/6965#issuecomment-2915796022
+get_home() {
+    if [ -n "${HOME:-}" ]; then
+        echo "HOME"
```

Comment:
Are you intending to return the literal string "HOME" here and not "$HOME"?

Reply:
oh, nope!